contracts/src/periphery/cert/EigenDACertTypes.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.sol";
+
+/// @title EigenDACertTypes
+/// @notice This library defines the types for each EigenDA certificate version.
+/// @dev It is required that RBN be located in positions 64:96 (padded) in the ABI encoded certificate.
```

Comment:
This is wrong, its really in position 32:64.
I realized why it shows up at position 64:96 in your tests; thats because `VerifyDACert(bytes calldata abiEncodedCert)` itself requires the bytes to be abi.encoded, and abi encoded bytes is a dynamic struct, so is pushed to the tail, and the head just becomes a pointer offset into the tail.
So basically what we have is abi.Encoding(abi.encoding(cert)).
RBN is in position 32:64 of abi.encoding(cert), but in position 64:96 of abi.encoding(abi.encoding(cert))